src/dune_engine/build_system.ml

```diff
@@ -532,10 +532,10 @@ end = struct
     in
     (* Note: we do not run the below in parallel with the above: if we fail to
        compute action execution parameters, we have no use for the action and
-       might as well fail early, skipping unnecessary dependencies. The function
-       [Source_tree.execution_parameters_of_dir] is memoized, and the result is
-       not expected to change often, so we do not sacrifice too much performance
-       here by executing it sequentially. *)
+       might as well fail early, skipping unnecessary dependencies. The
+       function [(Build_config.get ()).execution_parameters] is likely
```

Comment:
What do you mean by likely memoized?

Reply:
Just that it doesn't have to be memoized. For example, it could be the constant function if the execution parameters never vary.